### PR TITLE
finalize CHANGES

### PR DESCRIPTION
**Pull request by orbeckst**
- release 0.7.0 date in CHANGES
- update setup.py (twine check indicated missing long_description_content_type)
- for #105

### CODE REVIEW

**Comment by orbeckst** [comment]
Failed with
```
Fatal error:
There are 1 perturbed non-bonded pair interactions beyond the pair-list cutoff
of 1.039 nm, which is not supported. This can happen because the system is
unstable or because intra-molecular interactions at long distances are
excluded. If the latter is the case, you can try to increase nstlist or rlist
to avoid this.The error is likely triggered by the use of couple-intramol=no
and the maximal distance in the decoupled molecule exceeding rlist.
```
so we can run into the GROMACS exclusions bug in our tests. We need to increase rlist to guarantee tests to pass.

**Comment by orbeckst** [comment]
Given that the tests pass most of the time I am noy going to change them right now, if needed rerun a failing test. Will raise an issue.

File: CHANGES [hunk 1]
```diff
@@ -5,7 +5,7 @@ CHANGES for MDPOW
 Add summary of changes for each release. Use ISO dates. Reference
 GitHub issues numbers and PR numbers.
 
-2021-07-xx    0.7.0
+2021-07-29    0.7.0
```
Comment (orbeckst) [inline CHANGES:8]:
```suggestion
2021-08-02    0.7.0
```

File: doc/sphinx/source/conf.py [hunk 2]
```diff
@@ -137,12 +136,12 @@
 
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
-#html_logo = None
+html_logo = "_static/mdpow-logo.png"
```
Comment (orbeckst) [inline doc/sphinx/source/conf.py:139]: look at the docs under status : we hav a logo now

**Review by ALescoulie — APPROVED** [review]
Looks good, the updates to the docs will defiantly be a plus. With the scripts fixed it should make the terminal workflow compatible with modern python.

**Review by iorga — APPROVED** [review]
Looks good to me. Thanks for the hard work !

**Review by VOD555 — APPROVED** [review]

**Comment by orbeckst** [comment]
Thank you all for having a look, much appreciated.

**Pull request merged**